mmv1/templates/terraform/examples/vpc_access_connector_shared_vpc.tf.erb

```diff
@@ -0,0 +1,23 @@
+resource "google_vpc_access_connector" "connector" {
```

Comment:
I don't think this sample configures the firewall rules correctly?
https://cloud.google.com/vpc/docs/configure-serverless-vpc-access#firewall-rules

This is a prerequisite to using this feature as is my understanding.
@rileykarson

Reply:
It is like configuring VPNs. You need additional firewall rules to complete it. It is left to the end users to determine those rules.

Reply:
Either approach is fine imo. Not configuring the rules matches the precedent of most basic examples, which are often the barest possible config that will provision, although configuring them could be nice because then users can reference the example for that extremely common config.